__test__/update.test.js

```diff
@@ -0,0 +1,34 @@
+import addTask from '../src/modules/add.js';
+import displayTasks from '../src/modules/display.js';
```

Comment:
- [ ] Please add a test for editing the task as well

Reply:
Test to update the _description_ and  _checkbox_ status were splited from `update.test.js`. Both methods are inside the displayTasks function.

Checkbox: https://github.com/Camilovelag/Todo_list/blob/testing-2/__test__/checkbox.test.js
Description: https://github.com/Camilovelag/Todo_list/blob/testing-2/__test__/description.test.js